en/docs/install-and-setup/upgrading-wso2-api-manager/310-to-410/upgrading-from-310-to-410.md

```diff
@@ -505,23 +505,36 @@ Follow the instruction below to upgrade the Identity component in WSO2 API Manag
 
 7. Copy the `org.wso2.carbon.is.migration-x.x.x.jar` from the `<IS_MIGRATION_TOOL_HOME>/dropins` directory to the `<API-M_4.1.0_HOME>/repository/components/dropins` directory.
 
-8. Add below configuration  to `<API-M_4.1.0_HOME>/repository/conf/deployment.toml` to disable group role separation.
+    !!! important
+            In WSO2 Identity Server 5.11.0, groups include user store roles and roles include internal roles. Therefore, from IS 5.11.0 onwards, we cannot have the same admin role in both primary and internal user domains. If the same admin role exists in both UM domains of your older version, you have to rename the internal admin role into a different role name. To do that, you have to follow the below steps.
```

Comment:
Whether there are same admin roles in both primary and internal or not, user has to add following config

```
[super_admin]
create_admin_account = false 
```

shall we highlight this also?

Reply:
Fixed with https://github.com/wso2/docs-apim/pull/6092/commits/4c3bfdcd44df9cb56c94cc61ba22c39c2e214938

Reply:
This configuration is not needed in the APIM only identity migration step. Therefore removed it.